react/features/recording/components/Recording/web/StartRecordingDialog.js

```diff
@@ -19,6 +19,31 @@ import StartRecordingDialogContent from '../StartRecordingDialogContent';
  * @augments Component
  */
 class StartRecordingDialog extends AbstractStartRecordingDialog {
+
+    isStartRecordingDisabled: () => boolean;
+
+    /**
+     * Disables start recording button.
+     *
+     * @returns {boolean}
+     */
+    isStartRecordingDisabled() {
+        const { _fileRecordingsServiceEnabled, _isDropboxEnabled } = this.props;
+        const { isTokenValid, isValidating } = this.state;
+
+        // Start button is disabled if recording service is only shown;
+        // When validating dropbox token, if that is not enabled, we either always
+        // show the start button or, if just dropbox is enabled, start button
+        // is available when there is token.
+        if (_fileRecordingsServiceEnabled) {
+            return isValidating;
+        } else if (_isDropboxEnabled) {
+            return !isTokenValid;
+        }
+
+        return true;
```

Comment:
So this was returning false before and now it returns true... ?

Reply:
This is now updated.

Reply:
It's technically unreachable code, but better be safe.